llap-server/src/java/org/apache/hadoop/hive/llap/metrics/LlapDaemonExecutorInfo.java

```diff
@@ -64,6 +64,8 @@
   ExecutorFallOffNumCompletedFragments("Number of completed fragments w.r.t falloff values"),
   AverageQueueTime("Average queue time for tasks"),
   AverageResponseTime("Average response time for successful tasks"),
+  NumExecutorsEnabled("The number of executor threads enabled"),
+  WaitQueueSizeEnabled("Size of wait queue enabled"),
```

Comment:
Not sure of "enabled" is right in the context of a wait queue. It would just remove "enabled".

Reply:
I've done some renaming, let me know what do you think.